_data/design/office-365-guardrails.yml

```diff
@@ -0,0 +1,39 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      The purpose of the guardrails is to ensure that departments and agencies
+      are implementing a preliminary baseline set of controls within their
+      Microsoft O365 cloud-based environment.
+    fr: >-
+      Le présent projet met l’accent sur un ensemble préliminaire de mesures de
+      sécurité de base pour veiller à ce que l’environnement des services
+      d’informatique en nuage.
+designTypes:
+  - type:
+      en: Security & Privacy
+      fr: Sécurité et confidentialité
+homepageURL:
+  en: 'https://canada-ca.github.io/cloud-guardrails-O365/'
+  fr: 'https://canada-ca.github.io/cloud-guardrails-O365/'
+licences:
+  - URL:
+      en: 'https://github.com/canada-ca/cloud-guardrails-O365/blob/master/LICENSE'
+      fr: 'https://github.com/canada-ca/cloud-guardrails-O365/blob/master/LICENSE'
+    spdxID: MIT
+    level:
+      en: Main license
+      fr: !<tag:yaml.org,2002:js/undefined> ''
```

Comment:
Unrelated bug?

Reply:
It was #1080 that created the PR as a test with legitimate data. Kept it as an example without editing the value.